blocks/rich-text/index.js

```diff
@@ -387,6 +400,19 @@ export class RichText extends Component {
 		this.context.onCreateUndoLevel();
 	}
 
+	getContainerNode() {
+		// Find the parent "relative" or "absolute" positioned container
+		const findRelativeParent = ( node ) => {
+			const style = window.getComputedStyle( node );
```

Comment:
As mentioned earlier, totally optional as it's existing code, but might be good to move this to the top as a browser dependency. Just a personal preference.

Reply:
That makes sense. Made the change in https://github.com/WordPress/gutenberg/pull/5794/commits/7adab327c2327d890fce2a3c33b978f323950f22.